charts/newrelic-logging/templates/daemonset.yaml

```diff
@@ -105,6 +105,9 @@ spec:
           resources:
 {{ toYaml .Values.resources | indent 12 }}
           {{- end }}
+          {{- if .Values.securityContext }}
+          securityContext: {{ toYaml .Values.securityContext | nindent 12 }}
+          {{- end }}
```

Comment:
Chart implementations should use newrelic's common library:

https://github.com/newrelic/helm-charts/blob/d41364a102659445b03938f2a6f97d39deb38600/library/CHART-TEMPLATE/templates/deployment.yaml#L26-L29